app/modules/topics/saga/task/index.js

```diff
@@ -16,7 +16,7 @@ const taskSaga = function* (): Generator<*, *, *> {
     takeEvery(t.EDIT, editSaga),
     takeEvery(t.GET, getSaga),
     takeEvery(t.GET_ALL_BY_USERID, getAllSaga),
-    takeEvery(t.REMOVE, removeSaga),
+    takeLatest(t.REMOVE, removeSaga),
```

Comment:
Why did this change? `takeLatest` will fail on subsequent dispatches with different arguments. See #78.